examples/1-eks-cluster-with-new-vpc/main.tf

```diff
@@ -105,6 +231,9 @@ module "aws-eks-accelerator-for-terraform" {
   zone              = local.zone
   terraform_version = local.terraform_version
 
+  # KMS key
+  kms_key_policy = data.aws_iam_policy_document.eks_key.json
```

Comment:
Is this policy for `aws_vpc` module or `eks` module?

Reply:
eks